File: app/src/main/java/io/github/engsergiu/react/AboutActivity.java
package io.github.engsergiu.react;


import android.content.Intent;
import android.graphics.Color;
import android.net.Uri;
import android.os.Bundle;
import android.support.design.widget.FloatingActionButton;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.Toolbar;
import android.view.View;


public class AboutActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_about);
        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
        setSupportActionBar(toolbar);

        FloatingActionButton fab = (FloatingActionButton) findViewById(R.id.fab);
        fab.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                mailto("44aaff@gmail.com");
            }
        });
        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
        this.getWindow().setStatusBarColor(Color.parseColor("#BA1AEE"));
    }


    private void mailto(String email) {
        Intent launchBrowser = new Intent(Intent.ACTION_SENDTO, Uri.fromParts(
                "mailto",email, null));
        startActivity(launchBrowser);
    }
}

File: app/src/main/java/io/github/engsergiu/react/AboutActivity.kt
package io.github.engsergiu.react

import android.content.Intent
import android.graphics.Color
import android.net.Uri
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.appcompat.widget.Toolbar
import com.google.android.material.floatingactionbutton.FloatingActionButton

/**
 * "About" activity page.
 * @author Sergiu
 */
class AboutActivity : AppCompatActivity() {

	override fun onCreate(savedInstanceState: Bundle?) {
		super.onCreate(savedInstanceState)
		setContentView(R.layout.activity_about)
		setSupportActionBar(findViewById(R.id.toolbar))

		findViewById<FloatingActionButton>(R.id.fab).apply {
			setOnClickListener { mailto() }
		}
		supportActionBar!!.setDisplayHomeAsUpEnabled(true)
		this.window.statusBarColor = Color.parseColor("#BA1AEE")
	}

	private fun mailto() = startActivity(Intent(
			Intent.ACTION_SENDTO,
			Uri.fromParts("mailto", "44aaff@gmail.com", null)
	))
}